docs/docs/build/examples/btc_swap/index.md

```diff
@@ -1,34 +1,170 @@
+- Click on the 'Fulfil order' button, and enter the amount you wish to fulfil.
+- You will be prompted to sign the transaction in your wallet.
+
+If you now switch back to the account you used to create the order, you will see that amount available has decreased. If the order has been fulfilled completely, it will no longer be shown under the 'Sell' tab.
+
+![fill-order-form](fill-order-form.png)
+
+#### Fulfilling an ERC20 -> BTC order (i.e., buying BTC)
+
+- Switch to the account you created for fulfilling orders.
+- In the 'Buy' table find the swap you created with your selling account offering BTC (as the seller) in exchange for an ERC20.
+- Click on 'Fill Order.' The 'Pay with' and 'You will receive' fields are non-editable.
+- Enter a valid testnet BTC address.
+- Submit the form.
+
+The 'Fulfil order' button will now be disabled, and you will see a countdown next to it. This is how long the other party in the swap has to send you the BTC. Once the timer reaches 0, you will be able to cancel the order. This does not happen automatically: the other party will still be able to send the BTC unless you cancel the order.
+
+- Switch back to your selling account.
+- You will now see an 'Accepted BTC Orders' table under the 'Sell' tab.
+- Find the order you created in this table, and click on the 'Complete order' button.
+- Using a BTC wallet connected to the BTC testnet, send the BTC amount to the address shown. You can scan the QR code to use a mobile wallet.
+- You will be able to complete the order as soon as one confirmation has been received.
+  - If you close the modal or navigate away from the app, clicking on the 'Complete Order' button again will show you the current status of the transaction.
+
+#### Fulfilling a BTC -> ERC20 order (i.e., selling BTC)
+
+- Switch to the account you created for fulfilling orders.
+- In the 'Buy' table find the swap you created with your selling account offering an ERC20 token (as the seller) in exchange for BTC.
+- Click on 'Fill Order.' The 'Pay with' and 'You will receive' fields are non-editable.
+- Submit the form.
+
+You will now see an 'Accepted BTC Orders' table under the buy tab.
+
+- Find the order in the table, and click on the 'Complete Order' button.
+- Using a BTC wallet connected to the BTC testnet, send the BTC amount to the address shown. You can scan the QR code to use a mobile wallet.
+- You will be able to complete the order as soon as one confirmation has been received.
+  - If you close the modal or navigate away from the app, clicking on the 'Complete Order' button again will show you the current status of the transaction.
+
+If you switch back to your selling account before completing the order, you will see the order in the 'Accepted BTC Orders' table under the 'Sell' tab. If the order is not completed by the buyer within the allowed time, you will be able to cancel the order.
+
+![complete-order](complete-order.png)
+
+## Demo Code
+
+TODO: Code is only partially documented/
```

Comment:
Should we delete this line?

Reply:
Yes—its' true, but doesn't need to be commented :)